test/workers/audio_callback_worker_test.rb

```diff
@@ -16,47 +16,46 @@
 
   after(:each) { Shoryuken::Logging.logger.level = Logger::INFO }
 
-  # {
-  #   "id":987654,
-  #   "path":"public/audio_files/987654",
-  #   "name":"test_file.mp3",
-  #   "duration":1045,
-  #   "size":16705871,
-  #   "format":"mp3",
-  #   "bitrate":128000,
-  #   "frequency":44100,
-  #   "channels":2,
-  #   "layout":"stereo",
-  #   "downloaded":true,
-  #   "valid":true,
-  #   "processed":true
-  # }
-
   def perform(attrs = {})
     defaults = {
       id: audio.id,
+      name: 'test.mp3',
+      size: 1234,
+      mime: 'audio/mpeg',
       downloaded: true,
       valid: true,
       processed: true,
-      name: 'test.mp2'
+      error: nil,
```

Comment:
ending with an extra comma? what are they, growing on trees? 🌴

Reply:
What is this, IE6?